### ISSUE TITLE
A11y_.NET Core_Recipes App My Recipes New_New Recipe_Voiceover: After changing the value for review, voice over is not announcing new value its announcing incorrect rating value. 

### ISSUE THREAD

**Issue by Swati1700** [dotnet/maui#15634]
### Description

[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NET Core[MAUI]-MAC-June2023; #.NET Core; #WCAG1.3.1; #Element:Label; #ScreenReader; #Voiceover; #MAC;#Closed;

### Environment Details:​
Application Name: Rachel's Recipe
macOS: Ventura 13.3
Screen Reader: Voiceover

### Steps to Reproduce

1.Install and open ".NET MAUI" app.
2.TAB till "My Recipes" and select.
3.TAB till "New" button and select.
4."New Recipe" Screen will open.
5. Start Voiceover.
6. TAB till Slider and increment, decrement button and do changes in review.
7. TAB till rating displayed above the slider.
8. Observe voiceover is not announcing correct rating after updating the review rating.

### Actual Results:
In "New Recipe" screen voice over is announcing incorrect information. After changing the value for review, voice over is not announcing new value its announcing incorrect rating value like "10.0 out of 10". 

**Similar issue is also reproducing in the below scenarios:
Issue: Recipes App My Recipes New_New Recipe**

1. Install and open ".NET MAUI" app.
2. TAB till "My Recipes" and select.
3. TAB till any existing recipe and select.
4. "Recipe" Screen will open.
5. TAB till "Edit" button and select.
6. Enter text in all entry fields.
7. Start voiceover.
8. TAB till Rating value.
9. Observe voiceover is not announcing correct rating after updating the review rating.

 ### Expected Results:
After changing the value for review, voice over should announce new updated rating value like "7.3 out of 10". 

### User Impact:
Voiceover users will face difficulties while navigating to the rating value if voiceover will announce incorrect announcement of rating.

### Attachment:

https://github.com/dotnet/maui/assets/86710888/1e59e4f9-9943-42a7-b2ef-3dda26673087

**Note**: "As above bug is discussed with PWD user, also Severity and MAS reference is given as per discussion with them."

### Link to public reproduction project repository

Test

### Version with bug

7.0.49

### Last version that worked well

6.0

### Affected platforms

macOS

### Affected platform versions

Ventura 13.3

**Comment by rachelkang** [comment]
Hi, @Swati1700 - could you please provide clarification and/or video for the expected vs. actual behaviors?

To my understanding, we already match the expected behavior mentioned here already aligns with the actual behavior.
When updating the slider, VoiceOVer announces something like "73%, Rating increased to 7.3 out of 10 stars"

The "73%" is the slider value as a percentage. This behavior is external, and is announced as a consequence of the MacCatalyst platform behavior. The latter "Rating decreased to 7.3 out of 10 stars" is what we've added to reflect the expected behavior you describe.

Please let me know if I'm missing something here. Thanks!

If it's helpful, the way the slider is navigated with VoiceOver is as follows:
Tab key or Caps+Down key until screen reader focus is on the slider.
Ctrl+Option+Shift+Down key to interact with slider
Caps+Left/Right to move slider up/down --> hear VoiceOver announce rating as expected

**Comment by Swati1700** [comment]
Added the attachment in the bug.

**Comment by Swati1700** [comment]
Verified the bug in the latest app Recipes Version 2(2). Issue is fixed and does not repro. Hence closing the bug, please find the closing attachment.

https://github.com/dotnet/maui/assets/86710888/3ed2bf0c-e378-46db-b660-65d693cbcff5
